docs/topics/null-safety.md

```diff
@@ -143,6 +143,17 @@ is `null`, the assignment is skipped and the expression on the right is not eval
 person?.department?.head = managersPool.getManager()
 ```
 
+## Nullable receiver
+
+Functions could be extended to use a [nullable receiver](extensions.md#nullable-receiver).
```

Comment:
“Functions could be extended” is a confusing phrasing. How about “Extension functions can be defined on a nullable receiver”?